charts/topolvm/templates/node/daemonset.yaml

```diff
@@ -24,8 +24,8 @@ spec:
         {{- with .Values.node.podLabels }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
-      {{- if and .Values.node.metrics.enabled .Values.node.metrics.annotations }}
-      annotations: {{ toYaml .Values.node.metrics.annotations | nindent 8 }}
+      {{- if and .Values.node.csi.metrics.enabled .Values.node.csi.metrics.annotations }}
```

Comment:
breaking change with previous chart versions, needs a major version

Reply:
Agree added ! and Breaking Change to the commits to ensure this was communicated

Reply:
Thats nice! But maybe we should think about wether we need to move this CSI section at all? What benefit does it bring or is it necessary? If not, maybe we can avoid the breaking change alltogether?

Reply:
Might have been over agressive but these options don't apply to the pod only the csi container so I was attempting to not introduce ambiguity with the changes to move lvmd into the pod

Reply:
Ah understood, makes sense to me :) . Then lets wait for other maintainers to give feedback on if this will be accepted.